docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -132,6 +132,7 @@ following works:
 - github.com/shirou/gopsutil [BSD 3-Clause Clear License](https://github.com/shirou/gopsutil/blob/master/LICENSE)
 - github.com/sirupsen/logrus [MIT License](https://github.com/sirupsen/logrus/blob/master/LICENSE)
 - github.com/soniah/gosnmp [BSD 2-Clause "Simplified" License](https://github.com/soniah/gosnmp/blob/master/LICENSE)
+- github.com/stefa975/go-http-digest-auth-client [BSD 2-Clause "Simplified" License](https://github.com/stefa975/go-http-digest-auth-client/blob/master/LICENSE.md)
```

Comment:
Any reason you don't use the "upstream" version at https://github.com/xinsnake/go-http-digest-auth-client?

Reply:
Please try to merge your fork of that library with the upstream version!

Reply:
I did a pull Request to the upstream project, but he replied it's not maintained any more.
I'll try to find som other library.

Reply:
The alternative is you merge your repo with the upstream one and maintain it. It's ok for me, but we need a good reason here. :-)